berkeley-mobile/Libraries/LibraryDetailViewController.swift

```diff
@@ -39,7 +39,19 @@ class LibraryDetailViewController: SearchDrawerViewController {
     /// Opens `kBookingURL` in Safari.
     @objc private func bookButtonClicked(sender: UIButton) {
         guard let url = URL(string: kBookingURL) else { return }
-        UIApplication.shared.open(url)
+        
+        presentAlertWithTitle(title: "Are you sure you want to open Safari?", message: "Berkeley Mobile wants to open Libcal to book a study room", options: "Cancel", "Yes") { (option) in
```

Comment:
same as above, change from "are you sure"? also consider unifying this switch behavior because it's the most common case